### DISCUSSION TITLE
Customize tags list order

### DISCUSSION THREAD

**Discussion by gpsblues** [GetPublii/Publii#1372] [Help] [answered]
Hi,

Is there a way to sort a list of extracted tags with {{getTags}}?
I tried using {{reverse}} and {{orderby}} but I was unable to use the "collection" parameter.

Thankyou
```

{{#getTags (concatenate @config.custom.listingTag) "" ""}}
<p>{{name}}</p>
{{/getTags}}


```

**Comment by dziudek** [comment]
@gpsblues as it is stated in the helper description:

```
 * Tags are ordered by the ID order in the string.
```

So you must provide the IDs in `@config.custom.listingTag` in a specific order or try to use subexpressions to apply concatenate and orderby/reverse operations  like:

```
(concatenate (orderby @config.custom.listingTag 'id' 'desc'))
```

**Reply by gpsblues** [reply, marked as answer]
I can't use {{orderby}} correctly, but {{reverse}} acts directly on the string and I get the desired result.
So I use {{reverse}} before {{getTags}}.


```
{{reverse @config.custom.listingTag}}
{{#getTags (concatenate @config.cu@config.stom.listingTag) "" ""}}
...
{{/getTags}
```

@config.listingTag is obtained from a "Tags dropdown" field with the option multiple:true
